onnxruntime/python/tools/quantization/notebooks/bert/Bert-GLUE_OnnxRuntime_quantization.ipynb

```diff
@@ -1,971 +1,1041 @@
+        "\n",
+        "print('Evaluating ONNXRuntime full precision accuracy and performance:')\n",
+        "time_ort_model_evaluation('bert.opt.onnx', configs, tokenizer, \"onnx.opt\")\n",
+        "    \n",
+        "print('Evaluating ONNXRuntime quantization accuracy and performance:')\n",
+        "time_ort_model_evaluation('bert.opt.quant.onnx', configs, tokenizer, \"onnx.opt.quant\")"
+      ],
+      "execution_count": 13,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Evaluating ONNXRuntime full precision accuracy and performance:\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "/usr/local/lib/python3.7/dist-packages/transformers/data/processors/glue.py:175: FutureWarning: This processor will be removed from the library soon, preprocessing should be handled with the 🤗 Datasets library. You can have a look at this example script for pointers: https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py\n",
+            "  warnings.warn(DEPRECATION_WARNING.format(\"processor\"), FutureWarning)\n",
+            "Evaluating: 100%|██████████| 408/408 [02:39<00:00,  2.56it/s]\n",
+            "/usr/local/lib/python3.7/dist-packages/transformers/data/metrics/__init__.py:66: FutureWarning: This metric will be removed from the library soon, metrics should be handled with the 🤗 Datasets library. You can have a look at this example script for pointers: https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py\n",
+            "  warnings.warn(DEPRECATION_WARNING, FutureWarning)\n",
+            "/usr/local/lib/python3.7/dist-packages/transformers/data/metrics/__init__.py:42: FutureWarning: This metric will be removed from the library soon, metrics should be handled with the 🤗 Datasets library. You can have a look at this example script for pointers: https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py\n",
+            "  warnings.warn(DEPRECATION_WARNING, FutureWarning)\n",
+            "/usr/local/lib/python3.7/dist-packages/transformers/data/metrics/__init__.py:36: FutureWarning: This metric will be removed from the library soon, metrics should be handled with the 🤗 Datasets library. You can have a look at this example script for pointers: https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py\n",
+            "  warnings.warn(DEPRECATION_WARNING, FutureWarning)\n"
+          ],
+          "name": "stderr"
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "{'acc': 0.8602941176470589, 'f1': 0.9018932874354562, 'acc_and_f1': 0.8810937025412575}\n",
+            "Evaluate total time (seconds): 159.8\n",
```

Comment:
The perf number is not updated in the conclusion part. What machine are you using? It looks slow.

Reply:
Yes, good catch. I tested this notebook on Google Colaboratory, with their standard Python 3 Google Compute Engine backend (single CPU). But the performance metrics given are with the Azure VM: Standard E4ds_v4 (4 vcpus, 32 GiB memory). I think that is the reason for the significant decrease in speed.

I do not have access to multiple fast CPUs now. If you wish, you can retest this code on the original Azure VM, update the performance metrics and commit on this branch. Alternatively, I can update the performance metrics according to the Google Colab standard CPU runtime. What do you think we should do?